packages/test-run-controller/src/test-run-controller.ts

```diff
@@ -253,7 +253,8 @@ export class TestRunController extends PluggableModule implements ITestRunContro
                 worker.execute(
                     queuedTest.test,
                     queuedTest.parameters,
-                    this.config.envParameters
+                    this.config.envParameters,
+                    this.config.httpThrottle
```

Comment:
We have `runData` property in this.config.parameters. Not sure it's a good idea to make new argument for every config parameter that we will need.